app/pages/reset-password/new-password-form.spec.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import assert from 'assert';
+import { shallow, mount } from 'enzyme';
+import sinon from 'sinon';
+import NewPasswordForm from './new-password-form';
+
+describe('NewPasswordForm', () => {
+  let wrapper;
+
+  before(() => {
+    wrapper = shallow(<NewPasswordForm />);
+  });
+
+  it('renders a new password form for a user', () => {
+    assert.equal(wrapper.find('form').length, 1);
+  });
+
+  it('should call onSubmit when user submits a new password', () => {
+    const newPasswordSpy = sinon.spy();
+    wrapper = mount(<NewPasswordForm onSubmit={newPasswordSpy} />);
```

Comment:
Same comments here about not needing mount, when to setup the spy, and simulated the onSubmit using the form.